modules/phase_field/doc/content/source/auxkernels/PFFractureBoundsAux.md

```diff
@@ -0,0 +1,28 @@
+# PFFractureBoundsAux
+
+!syntax description /AuxKernels/PFFractureBoundsAux
+
+## Description
+
+`PFFractureBoundsAux` is used for setting bounds on phase field fracture variable. The upper bound is set as 1 by default. The lower bound is set as variable's old value in order to enforce irreversibility condition.
+
+Note that in order for these bounds to have an effect, the user has to specify the
+PETSc options `-snes_type vinewtonssls` or `-snes_type vinewtonrsls`. Both
+options emply semi-smooth non-linear solution algorithms to solve the
```

Comment:
`vinewtonssls` is a semi-smooth Newton method, but `vinewtonrsls` does not employ the semi-smooth algorithm.  `vinewtonrsls` is the reduced space algorithm.

Reply:
https://arxiv.org/pdf/math/0307305.pdf